docs/automated_evaluation/automated_evaluation_overview.md

```diff
@@ -2,7 +2,17 @@
 
 ## Overview
 
-To score each team's system, an semi-automated system is used. A docker container is created for each team with the ARIAC environment pre-installed. The docker container also includes VNC in order to display the gazebo environment while it is run. To properly build and install the competitor's code onto the container each team must submit a configuration file using the following template. The yaml configuration file should be submitted to the Google Drive folder for your team.
+To score each teams system, an semi-automated system is used. A docker container is created for each team with the ARIAC environment pre-installed.
+we will run a series of automated evaluations and trial evaluations. To properly build and install the competitor's code onto the container each team must submit a configuration file using the following template. The yaml configuration file should be submitted to the Google Drive folder for your team..
```

Comment:
I would remove this sentence:

we will run a series of automated evaluations and trial evaluations